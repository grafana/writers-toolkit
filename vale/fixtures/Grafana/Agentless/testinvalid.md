Don't use:

- agentless deployment
- agentless solution
