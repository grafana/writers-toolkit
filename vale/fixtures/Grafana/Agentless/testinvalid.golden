testinvalid.md:3:3:Grafana.Agentless:Grafana Agent has been replaced by Grafana Alloy, so you shouldn't use agent-based terminology. If you're talking about why and how to send signals directly from an application to Grafana Cloud, prefer no-collector to agentless. This is consistent with [OTel documentation](https://opentelemetry.io/docs/collector/deployment/no-collector/). 
testinvalid.md:4:3:Grafana.Agentless:Grafana Agent has been replaced by Grafana Alloy, so you shouldn't use agent-based terminology. If you're talking about why and how to send signals directly from an application to Grafana Cloud, prefer no-collector to agentless. This is consistent with [OTel documentation](https://opentelemetry.io/docs/collector/deployment/no-collector/). 
